components/uniset/src/lib.rs

```diff
@@ -2,6 +2,9 @@
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
+//! *** Note: DO NOT USE THESE APIs FOR NOW. ****
+//!  Performance improvements and other fixes are still needed on this component.
```

Comment:
Uniset is okay to use, just not the "props" module.  Uniset is Evan's work and it was already shipped in 0.1.